cmd/mayactl/app/command/volume.go

```diff
@@ -69,13 +69,13 @@ func NewVolumeInfo(URL string, volname string, namespace string) (volInfo *Volum
 	}
 	if resp != nil && resp.StatusCode != 200 {
 		if resp.StatusCode == 500 {
-			fmt.Printf("Volume: %s not found at M_API server\n", volname)
+			fmt.Printf("Volume: %s not found at namespace: %q\n", volname, namespace)
 			err = util.InternalServerError
 		} else if resp.StatusCode == 503 {
 			fmt.Println("M_API server not reachable")
```

Comment:
Can you fix this also to print what is the IP address used to connect. Rename --> "M_API server" to maya apiservice

Reply:
sure sir